github-api/src/main/java/org/eclipse/egit/github/core/service/PullRequestService.java

```diff
@@ -65,7 +65,13 @@
 	 */
 	public static final String PR_BODY = "body"; //$NON-NLS-1$
 
-	/**
+    /**
```

Comment:
Code style: This module uses different indentation rules than the rest of the project. Please follow them. I think here it's indented by tabs.